recipe/meta.yaml

```
@@ -1,47 +1,91 @@
+{% set name = "spreg" %}
 {% set version = "1.2.4" %}
 
 package:
-  name: spreg
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/s/spreg/spreg-{{ version }}.tar.gz
-  sha256: c2d1f243dedf53e3380c83fcd93bbd33339e20fb187486dc7ce7ebab7c21bc76
+  url: https://github.com/pysal/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 2e6a46513d92820b020a3a96b002c4138364c0b8998386f3952dee9f2c6809db
+  patches:
+    - patches/0000-add_numpy_import.patch
+    - patches/0001-remove-relative-imports-in-tests.patch
+    - patches/0002-numpy-arrays-to-stack-must-be-list-or-tuple.patch
+    - patches/0003-flatten-minimize-input.patch
 
 build:
-  number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+  number: 1
+  # skip s390x libpysal not available in 4.10
+  skip: True  # [py<37 or s390x]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
+  build:
+    - patch       # [not win]
+    - m2-patch    # [win]
   host:
-    - python >=3.7
+    - python
     - pip
     - setuptools
     - wheel
   run:
-    - python >=3.7
-    - libpysal >=4.0.0
+    - python
+    # libpysal: for py39 the version 4.5.1 will be used
+    #           while >=4.10 is available only for py>39
+    - libpysal
+    - libpysal >=4.10  # [py>39]
```

```suggestion
    - libpysal  # [py<310]
    - libpysal >=4.10  # [py>=310]
```
Wouldn't it be more clear?

---

done, thanks